lib/web_ui/lib/src/engine/compositor/painting.dart

```diff
@@ -0,0 +1,171 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of engine;
+
+/// The implementatin of [ui.Paint] used by the CanvasKit backend.
+class SkPaint implements ui.Paint {
+  SkPaint();
+
+  static const ui.Color _defaultPaintColor = ui.Color(0xFF000000);
+
+  ui.BlendMode get blendMode => _blendMode;
```

Comment:
The linter doesn't complain that all public components should have a docstring?

Reply:
The code in `engine` is in a private dart library `dart:_engine`

Reply:
Thanks! This reminded me to add `@override` annotations on these.